tensorflow/workspace.bzl

```diff
@@ -160,6 +160,7 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
     tf_http_archive(
         name = "mkl_dnn",
         build_file = clean_dep("//third_party/mkl_dnn:mkldnn.BUILD"),
+        system_build_file = clean_dep("//third_party/systemlibs:mkl_dnn.BUILD"),
```

Comment:
since the third_party/mkl_dnn/ dir exists, lets put the system one as `//third_party/mkl_dnn:BUILD.system`